src/AutorestSwift/Code Generator/SwiftGenerator.swift

```diff
@@ -34,6 +35,8 @@ class SwiftGenerator: CodeGenerator {
 
     let baseUrl: URL
 
+    let documentsUrl = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first
```

Comment:
This shouldn't be necessary. `main.swift` passes in the baseUrl as the documents directory

Reply:
Thanks. Removed.